commands/databases.go

```diff
@@ -78,6 +78,8 @@ There are a number of flags that customize the configuration, all of which are o
 	AddStringFlag(cmdDatabaseCreate, doctl.ArgDatabaseEngine, "", defaultDatabaseEngine, "The database engine to be used for the cluster. Possible values are: `pg` for PostgreSQL, `mysql`, `redis`, and `mongodb`.")
 	AddStringFlag(cmdDatabaseCreate, doctl.ArgVersion, "", "", "The database engine version, e.g. 14 for PostgreSQL version 14")
 	AddStringFlag(cmdDatabaseCreate, doctl.ArgPrivateNetworkUUID, "", "", "The UUID of a VPC to create the database cluster in; the default VPC for the region will be used if excluded")
+	AddStringFlag(cmdDatabaseCreate, doctl.ArgDatabaseRestoreFromCluster, "", "", "The name of an existing database cluster from which the backup will be restored.")
+	AddStringFlag(cmdDatabaseCreate, doctl.ArgDatabaseRestoreFromTimestamp, "", "", "The timestamp of an existing database cluster backup in ISO8601 combined date and time format. The most recent backup will be used if excluded.")
```

Comment:
Just a note about doclt (not your change).

Unfortunately, doctl lists backups with a different date format. So where this command expects a value like `2023-02-01T17:32:15Z` (which is what the API returns), doctl displays:

```
Size in Gigabytes    Created At
0.039608             2023-02-01 17:32:15 +0000 UTC
0.039649             2023-02-02 17:32:08 +0000 UTC
0.039690             2023-02-03 17:32:15 +0000 UTC
```

Maybe we could add a doctl config setting that would return all dates in ISO8601 format at some point.

Reply:
It might actually be a good idea to accept the format doctl outputs and convert it to ISO8601 for the user.

Reply:
Oh good catch! Will accept in UTC format (`2023-02-01 17:32:15 +0000 UTC`) and will convert to ISO861 to pass on to API. Will add to the next commit.